docs/slack.rst

```diff
@@ -53,7 +53,16 @@ Chances are we have a large group of folks in your neck of the woods,
 so join up and chat with them.
 They're a great way to get in touch with your local meetup organizer and community.
 
-Social Rules
+Rules
+-----
+
+We have a few hard and fast rules that we ask people to follow on our Slack:
+
+- **No Direct Pitches** - If you’re coming here *only* to make a sales pitch, please don’t. If you just put a link in the chat without any context or conversation, we will delete it.
```

Comment:
I think this should be split into two different rules:

 **No Direct Pitches** - If you’re coming here *only* to make a sales pitch, please don’t. We will delete them.

**No links without context** - If you just put a link in the chat without any context or conversation, we will delete it.

(I think this better fits the kind of spam that I've worked through, e.g. the miscellaneous "here's a poll" without context, e.g. https://trello.com/c/GgsMBN3q, https://trello.com/c/8XSRmVqs )